tools/rostopic/src/rostopic/__init__.py

```diff
@@ -120,7 +121,8 @@ def callback_hz(self, m):
         if self.filter_expr is not None and not self.filter_expr(m):
             return
         with self.lock:
-            curr_rostime = rospy.get_rostime()
+            curr_rostime = rospy.get_rostime() if not self.use_wtime else \
+                    rospy.Time.from_sec(time.time())
```

Comment:
Please indent the wrapped line with only +4 (not with +8 according to PEP8).

Same a few lines below.